Fix compile, passing NULL isn't an integer for initializing a bool.

From upstream revision 11469.

diff -r a9e748858e58 -r d77c7249db51 src/profiler/SoProfilerVisualizeKit.cpp
--- src/profiler/SoProfilerVisualizeKit.cpp	Mon Feb 16 13:09:45 2015 +0000
+++ src/profiler/SoProfilerVisualizeKit.cpp	Thu Feb 19 22:30:51 2015 +0100
@@ -150,7 +150,7 @@
   SO_KIT_ADD_FIELD(separatorsWithGLCaches, (NULL));
 
   this->separatorsWithGLCaches.setNum(0);
-  this->separatorsWithGLCaches.setDefault(NULL);
+  this->separatorsWithGLCaches.setDefault(FALSE);
   PRIVATE(this)->cacheSensor.reset(new SoFieldSensor(cacheSensorCB, this));
   PRIVATE(this)->cacheSensor->attach(&this->separatorsWithGLCaches);
   PRIVATE(this)->rootSensor.reset(new SoFieldSensor(rootChangedCB, this));
